style.css

```diff
@@ -127,3 +127,668 @@ li {
+  order: 1;
+  flex-grow: 0;
+  margin-left: -30px;
+}
+
+.btn3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  position: static;
+  width: 161px;
+  height: 48px;
+  left: 24px;
+  top: 395px;
+  background-color: #ff6b00;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: -30px;
+  margin-left: -28px;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  font-weight: bold;
+  font-size: 17px;
+  transition: solid 0s;
+  border-color: transparent;
+  text-align: center;
+}
+
+.border1 {
+  position: inherit;
+  width: 320px;
+  height: 1px;
+  left: 1px;
+  top: 10px;
+  border-bottom: 1px solid #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: 180px;
+  margin-left: -20px;
+}
+
+.lang {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: static;
+  width: 225.46px;
+  height: 281.46px;
+  left: 51.77px;
+  top: 120px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 40px;
+  color: #fff;
+}
+
+.main {
+  position: static;
+  width: 164px;
+  height: 24px;
+  left: 49.46px;
+  top: 8.73px;
+  font-family: "InaiMathi", "Tamil Sangam MN", "Nirmala UI", Latha, Bamini, Roboto, Noto, "Noto Sans Tamil", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: -350px;
+  margin-left: 20px;
+}
+
+.java {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 230px;
+  margin-left: 10px;
+}
+
+.ruby {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 60px;
+}
+
+.html {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 65px;
+}
+
+.css {
+  position: static;
+  width: 114px;
+  height: 24px;
+  left: 1px;
+  top: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 74px;
+}
+
+#fw {
```

Comment:
- [ ] The "Ruby on Rails" text does not match the rest of the design, you should consider setting up a different width so the full text can be in a single line.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143685111-d548b365-593f-42ae-a0f9-9d3a35069af0.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/143685122-3adb41da-5a14-4813-a209-928220eb3715.png)